.github/workflows/ci.yml

```diff
@@ -9,10 +9,69 @@ name: CI
 jobs:
   coding-guidelines:
     name: Coding Guidelines
+
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout
         uses: actions/checkout@master
 
       - name: Run friendsofphp/php-cs-fixer
-        run: php7.3 tools/php-cs-fixer fix --diff-format=udiff --dry-run --show-progress=dots --using-cache=no --verbose
+        run: php7.3 ./tools/php-cs-fixer fix --diff-format=udiff --dry-run --show-progress=dots --using-cache=no --verbose
+
+  lint-xml-configuration:
+    name: Lint XML Configuration
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Build Docker image
+        uses: ./.docker/lint-xml-configuration
+
+      - name: Lint XML configuration files
+        uses: ./.docker/lint-xml-configuration
+        with:
+          args: bash ./build/lint-configuration
+
+  tests:
+    name: Tests
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-binary:
+          - php7.1
+          - php7.2
+          - php7.3
+
+        dependencies:
+          - lowest
+          - highest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Install lowest dependencies with composer
+        if: matrix.dependencies == 'lowest'
+        run: ${{ matrix.php-binary }} ./tools/composer update --no-ansi --no-interaction --no-progress --no-suggest --prefer-lowest
+
+      - name: Install highest dependencies with composer
+        if: matrix.dependencies == 'highest'
+        run: ${{ matrix.php-binary }} ./tools/composer update --no-ansi --no-interaction --no-progress --no-suggest
+
+      - name: Run sanity check
+        run: PHP_BINARY=${{ matrix.php-binary }} bash ./build/sanity-check
+
+      - name: Run tests with phpunit
+        run: ${{ matrix.php-binary }} -d assert.exception=On -d zend.assertions=1 ./phpunit --coverage-clover=coverage.xml
+
+      - name: Download code coverage uploader for Codecov.io
+        run: curl -s https://codecov.io/bash -o codecov
+
+      - name: Send code coverage report to Codecov.io
+        run: bash codecov -t ${{ secrets.CODECOV_TOKEN }}
```

Comment:
@sebastianbergmann 

Looks like the `CODECOV_TOKEN` has not been set as repository secret, see https://github.com/sebastianbergmann/phpunit/pull/3833/checks?check_run_id=216468223.

Reply:
@sebastianbergmann 

Also see https://github.com/codecov/codecov-bash/pull/199 as an alternative.

Reply:
Just checked and `CODECOV_TOKEN` is set.

Reply:
@sebastianbergmann 

I think the challenge here is that the `CODECOV_TOKEN` is not available for a _pull request_ build. I assume that when you visit https://github.com/sebastianbergmann/phpunit/pull/3833/checks, that you also see a _push_ build?

![Screen Shot 2019-09-11 at 23 01 51](https://user-images.githubusercontent.com/605483/64735113-3056b200-d4e8-11e9-9874-2c8cd4b04efe.png)


Are the builds for the _push_ build passing?